src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Podobnie jak wyżej

Reply:
Lubię lombokowego pełnego konstruktora, bo przy każdym dodaniu/usunięciu pola z encji kompilator przypomina mi o zmianach we wszystkich niezbędnych miejscach.